code/controllers/subsystem/tts.dm

```diff
@@ -253,7 +272,7 @@ SUBSYSTEM_DEF(tts)
 
 	var/shell_scrubbed_input = tts_speech_filter(message)
 	shell_scrubbed_input = copytext(shell_scrubbed_input, 1, 300)
-	var/identifier = "[sha1(speaker + filter + shell_scrubbed_input)].[world.time]"
+	var/identifier = "[sha1(speaker + filter + shell_scrubbed_input + num2text(pitch) + num2text(silicon))].[world.time]"
```

Comment:
These should always be before the input, or else you could get a situation where someone is able to create a duplicate identifier when the values are different.
For example, if the pitch is set to 10 for a message with a content of "Hello" and another TTS request comes in with pitch 0 for a message of "Hello1", then it'll have the same identifier as the former message.